pinot-common/src/test/resources/sample_broker_config.properties

```diff
@@ -0,0 +1,29 @@
+pinot.zk.server=pinot-zookeeper-headless:2181
+pinot.shared.tls.internal.keystore.path=/home/tls-store/keystore-internal.jks
+pinot.shared.tls.internal.keystore.password=first-secret
+pinot.shared.tls.internal.keystore.type=JKS
+pinot.shared.tls.internal.truststore.path=/home/tls-store/truststore-internal.jks
+pinot.shared.tls.internal.truststore.password=second-secret
+pinot.shared.tls.internal.truststore.type=JKS
+pinot.shared.tls.internal.client.auth.enabled=true
+pinot.shared.tls.external.keystore.path=/home/tls-store/keystore-external.jks
+pinot.shared.tls.external.keystore.password=third-secret
+pinot.shared.tls.external.keystore.type=PKCS
+pinot.shared.tls.external.truststore.path=/home/tls-store/truststore-external.jks
+pinot.shared.tls.external.truststore.password=forth-secret
+pinot.shared.tls.external.truststore.type=PKCS
+pinot.shared.tls.external.client.auth.enabled=false
+pinot.broker.tls.ref=internal
+pinot.broker.client.access.protocols=internal,external
+pinot.broker.client.access.protocols.internal.protocol=https
+pinot.broker.client.access.protocols.internal.port=8099
+pinot.broker.client.access.protocols.internal.tls.ref=internal
```

Comment:
For reviewers: this line is the referencing part, where it is pointing to `pinot.shared.tls.internal`